scripts/qa_crowbarsetup.sh

```diff
@@ -1306,7 +1306,7 @@ EOF
 
     cd /tmp
 
-    local netfile="/opt/dell/chef/data_bags/crowbar/bc-template-network.json"
+    local netfile="/opt/dell/chef/data_bags/crowbar/template-network.json"
```

Comment:
:-1: this would be the wrong name on cloud4/5 so we would have to use iscloudver 6plus